src/js/controller/instream-adapter.js

```diff
@@ -126,6 +127,10 @@ var InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
         return this;
     };
 
+    function triggerPlayRejected() { 
+        _model.set('playRejected', true);
```

Comment:
Did you try setting this on the instream model?

Reply:
@robwalch  The reason I triggered it on the model was to hit the listener on the PlayDisplayIcon, which reacts to changes in state -- i.e. the goal was to get the play display icon to show.

Reply:
That uses the view-model which should get the change from instream's model when it's active. Try it please.